vms/platformvm/txs/fee/calculator_test.go

```diff
@@ -31,14 +31,11 @@ var (
 )
 
 func TestTxFees(t *testing.T) {
-	r := require.New(t)
-
 	type feeTests struct {
-		name          string
-		chainTime     time.Time
-		unsignedTx    func(t *testing.T) txs.UnsignedTx
-		expectedError error
-		checksF       func(*testing.T, *Calculator)
+		name       string
+		chainTime  time.Time
+		unsignedTx func(t *testing.T) txs.UnsignedTx
+		expected   uint64
```

Comment:
this is fine for now and I am in favour of having an `expected` attribute instead of the pair `expectedError` and `checksF`.
Note however that in next PRs, will go back to the pair since we will want to do more chacks than just fees (complexity for instance)